packages/relay/src/QueryRenderer.js

```diff
@@ -22,9 +23,10 @@ export default class App extends React.Component<Props> {
   renderRelayContainer = ({ error, props }: ReadyState) => {
     if (error) {
       return (
-        <View>
-          <Text style={styles.text}>{error.message}</Text>
-        </View>
+        <IllustrationWithInformation
+          illustrationName="Error"
+          text={error.message}
```

Comment:
In final version we will probably need some default error message instead of displaying exact error string (which can be quite confusing for user).
<img width="505" alt="screenshot 2019-01-22 at 15 03 29" src="https://user-images.githubusercontent.com/2660330/51540818-a920d500-1e57-11e9-829d-af2a66ee0ee9.png">

Or at least display some common "**Something went wrong**" header and then _error message_ underneath it with some small font and semi-transparent color.

But for now the current implementation is probably fine.

Reply:
OK, done

Reply:
<img width="333" alt="screenshot 2019-01-22 at 16 07 34" src="https://user-images.githubusercontent.com/858321/51544274-dbcecb80-1e5f-11e9-8c99-ba2249429cf2.png">